man1/aur.1

```diff
@@ -421,6 +421,15 @@ variable towards it:
     $ export PACMAN=/usr/local/bin/mypacman
 .EE
 .
+.SS Rebuilding packages against updated dependencies
```

Comment:
I might leave this section in `aur-build(1)`, if we want to keep it in `aur(1)` I'd write a specific command-line in the `EXAMPLES` section

Reply:
Moved to the `NOTES` section of `aur-build(1)`, or you have a better location?

Reply:
The location looks OK to me